src/loaders/assetLoader.js

```diff
@@ -28,7 +28,12 @@ module.exports = async function assetLoader() {
 
   const query = utils.getOptions(this) || {};
   const options = this.options[query.config] || {};
-  const config: Config = Object.assign({}, options, query);
+  const config: Config = Object.assign(
+    {},
+    { platform: 'ios', root: process.cwd() },
```

Comment:
`options` and `query` could be empty objects, so I had to introduce default `platform` and `root`. cc @satya164